Makefile

```diff
@@ -26,3 +26,6 @@ clean:
 	docker-compose run --rm node npm run clean
 	docker-compose kill
 	docker-compose rm -f
+
+format:
+	docker-compose run --rm node npm run format
```

Comment:
Where is the format defined?

Reply:
It's in [.prettierrc](https://github.com/silinternational/idp-profile-ui/blob/develop/.prettierrc). I added it awhile ago as there was no formatting. I should have done this then.